Update 2.meta-config.md (#1598)
vesoft-inc/nebula-docs: docs-2.0/5.configurations-and-logs/1.configurations/2.meta-config.md

| `port`                  | `9559`           | Specifies RPC daemon listening port of the Meta service. The external port for the Meta Service is predefined to `9559`. The internal port is predefined to `port + 1`, i.e., `9560`. Nebula Graph uses the internal port for multi-replica interactions. |
| `ws_ip`                   | `0.0.0.0`      | Specifies the IP address for the HTTP service.                                   |
| `ws_http_port`            | `19559`        | Specifies the port for the HTTP service.                                         |

> |`ws_storage_http_port`|`19779`| Specifies the Storage service listening port used by the HTTP protocol. It must be consistent with the `ws_http_port` in the Storage service configuration file. Only Standalone NebulaGraph needs to set this parameter.|

Reviewer: ```suggestion
|`ws_storage_http_port`|`19779`| Specifies the Storage service listening port used by the HTTP protocol. It must be consistent with the `ws_http_port` in the Storage service configuration file. This parameter is only to be set for Standalone NebulaGraph clusters.|
```

> |`ws_storage_http_port`|`19779`| Specifies the Storage service listening port used by the HTTP protocol. It must be consistent with the `ws_http_port` in the Storage service configuration file. This parameter is only to be set for Standalone NebulaGraph clusters.|

Reviewer 2: ```suggestion
|`ws_storage_http_port`|`19779`| Specifies the Storage service listening port used by the HTTP protocol. It must be consistent with the `ws_http_port` in the Storage service configuration file. This parameter only applies to standalone NebulaGraph.|
```
